src/closure.js

```diff
@@ -26,6 +54,19 @@ const cacheFunction = (cb) => {
   // If the returned function is invoked with arguments that it has already seen
   // then it should return the cached result and not invoke `cb` again.
   // `cb` should only ever be invoked once for a given set of arguments.
+  const arg = [];
+  const result = [];
+  return (unknown) => {
+    if (arg.indexOf(unknown) !== -1) {
+      return result[arg.indexOf(unknown)];
```

Comment:
Nice work. I really like indexOf. And I like that you used an array to cache your function call. I would rename this array to something more descriptive other than 'arg' just to make sure if someone was to re-read your code, they'd be able to understand exactly what 'arg' is.